src/main/java/hudson/plugins/deploy/NonBlankString.java

```diff
@@ -0,0 +1,26 @@
+package hudson.plugins.deploy;
```

Comment:
This seems overkill. Please keep changes to a minimum (a final String is IMHO enough in this case).